tools/gn

```diff
@@ -152,6 +152,8 @@ def to_gn_args(args):
         gn_args['dart_runtime_mode'] = 'profile'
     elif runtime_mode == 'dynamic_release':
```

Comment:
Are 'dynamic_profile' and 'dynamic_release' still used anywhere?

Reply:
No, I think these could be cleaned up here too if that's what you're asking :)

Reply:
Removed